WISHLIST.txt

```diff
@@ -14,4 +14,21 @@
 - craftable, upgradeable sentry bot. floats behind the player and shoots stuff
     - better batteries increase HP? or shields?
     - switchable weapon
-    - inventory/storage?
\ No newline at end of file
+    - inventory/storage?
+    
+    *AugiteSoul's ideas, edit them whenever you want ^^*
+- computercraft compat w/ MS peripherals, and auto-transit using those
+- tractor beams, forcingly move a ship, uses lots of RF
+- wide controllers, those can control machines you own in a large radius(disable all air locks?)
+- better GC boss defeat sound, death cry looks odd w/ some bosses
+- constricting attack for mummy?
+- bodies for amun?
+- basalt chest, more resistant, and is accessible only by the one who placed it 1st
+- mission MS, automatic, can get gases from atmosphere of bodies, or ores from bodies (asteroid belts mainly)
+- beamer, allows for safe descent on a body (will error when orbiting unavailable body)
+- vertical thrusters and directions, for giant 1-use rockets? (aka make stars follow y-axis when all thrusters do it)
+- energy golems, better stats than iron ones, uses RF, and defends robot villages

```

Comment:
Mekanism's Robit does that. It stores RF, and, when it's hurt, RF equivalent to damage taken is used. When it's depleted, it returns to it's pad, waiting for recharge. But the energy golem could just die, as it could be quite big for a charge pad :).